config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: ghcr.io/tinkerbell/cluster-api-provider-tinkerbell:dev
+      - image: ghcr.io/tinkerbell/cluster-api-provider-tinkerbell-amd64:dev
```

Comment:
Can we drop the `-amd64` again?

Reply:
@pjshah, when you build capt locally using "make docker-build", this file as well as the manager_pull_policy.yaml files below are updated automatically with these changes. You will need to revert these back.